static/app/components/modals/dashboardWidgetQuerySelectorModal.spec.tsx

```diff
@@ -88,18 +88,14 @@ describe('Modals -> AddDashboardWidgetModal', function () {
     MockApiClient.clearMockResponses();
   });
 
-  it('renders a single query selection when the widget only has one query', async function () {
-    const wrapper = mountModal({initialData, widget: mockWidget});
-    await tick();
-    expect(wrapper.find('StyledInput').length).toEqual(1);
-    expect(wrapper.find('StyledInput').props().value).toEqual(
-      'title:/organizations/:orgId/performance/summary/'
-    );
-    expect(wrapper.find('OpenInDiscoverButton').length).toEqual(1);
-    wrapper.unmount();
+  it('renders a single query selection when the widget only has one query', function () {
+    renderModal({initialData, widget: mockWidget});
+
+    screen.getByDisplayValue('title:/organizations/:orgId/performance/summary/');
+    screen.getByLabelText('Open in Discover');
```

Comment:
Not super important since these will just throw anyway, but using `expect().toBeInTheDocument()` makes it more clear exactly what the test is testing.